legal-api/src/legal_api/services/filings/validations/common_validations.py

```diff
@@ -197,15 +197,15 @@ def validate_pdf(file_key: str, file_key_path: str) -> Optional[list]:
     return None
 
 
-def validate_party_name(legal_type: str, party: dict, party_path: str) -> list:
+def validate_party_name(party: dict, party_path: str) -> list:
     """Validate party name."""
     msg = []
 
     custom_allowed_max_length = 20
     officer = party['officer']
     party_type = officer['partyType']
 
-    if party_type == 'person' and legal_type in [Business.LegalTypes.BCOMP.value, Business.LegalTypes.COOP.value]:
```

Comment:
@argush3 Removing this condition since we need to do colin sync all legal types for MVP